PIMbench/filter-by-key/PIM/filter.cpp

```diff
@@ -9,26 +9,27 @@
 #include <vector>
 #include <getopt.h>
 #include <cstdlib>
-#include <time.h>
+#include <time.h> 
 #include <algorithm>
 #include <chrono>
-#if defined(_OPENMP)
 #include <omp.h>
-#endif
```

Comment:
I think this is necessary to build successfully without openmp

Reply:
Sorry, this "#include <omp.h>" is actually no long needed, can you just help me remove that, I didn't notice that.